### ISSUE TITLE
Archived connectors are not removed from the platform

### ISSUE THREAD

**Issue by girarda** [airbytehq/airbyte#40661]
We recently archived source-google-analytics-v4. [The connector is not in the registry anymore](https://storage.googleapis.com/prod-airbyte-cloud-connector-metadata-service/registries/v0/cloud_registry.json), but it is still possible to create a new actor with that definition

<img width="2000" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNUtrQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--7a6549ea12a4c7fecec020ff073f155dd18c0c69/Screenshot%202024-07-01%20at%2010.25.55%20AM.png" alt="Screenshot 2024-07-01 at 10.25.55 AM.png" />

[Slack thread for context](https://airbytehq-team.slack.com/archives/C03A8CSANPQ/p1719841417134729)

**Comment by natikgadzhi** [comment]
Oh this is interesting. I've just looked up `source-punk-api` that @btkcodedev archived. It still shows up in the Marketplace. What's even more fun, up to date pipeline applied to it and updated it's deps, and _republished_ a new version: https://github.com/airbytehq/airbyte/pull/40972. 

The metadata for punk api clearly says `supportLevel: archived`, and `registries.cloud.enabled: false`. 

@evantahler when you worked on archiving, any last steps that we should make on the platform for connectors to be cleaned up from being available?

**Comment by natikgadzhi** [comment]
Oh, @girarda here's the guide to hiding connectors in the app: https://www.notion.so/Hiding-or-removing-a-connector-from-Cloud-0839d6c26445415d8aab5643a84db75e?pvs=4#a7a5570a1dcf4b03904e29a368a622e7 — I think we didn't do the "tombstone" part. 

I wonder what a good process for this looks like. @malikdiarra should we reach out to Compose folks on Slack, or do that ourselves? If it's just changing a column, Python / Marketplace can do it, but it would probably be good to automate so we don't forget? 

Should @airbytehq/dev-tooling look into adding that as a step of archiving pipeline?

**Comment by girarda** [comment]
@natikgadzhi the solution is to follow the 2-step process describe in notion. The issue I ran into is that the platform does not ingest the new support status if the connector is removed from the registry at the same time. I'm ok with "following the process" now that the documentation has been updated.

Closing, but sharing[ thread for additional context ](https://airbytehq-team.slack.com/archives/C0309PV2WCD/p1719885722240499?thread_ts=1717683118.132239&cid=C0309PV2WCD)
